src/component/Book.jsx

```diff
@@ -8,27 +8,27 @@ const Book = (props) => {
   } = props;
   return (
     <div className="book-container">
-      <div>
-        <p>{category}</p>
-        <h3>{title}</h3>
-        <small>{author}</small>
-        <div>
-          <button type="button">Comments</button>
+      <div className="book-details">
```

Comment:
- [ ] GReat job so far. please kindly change the background color ` #fafafa` around the cards so it looks like the zeplin design.
- [ ] please your progress update buttons should shift inside a little as they are not supposed to be aligned with the user photo place holder and the add book button.
- [ ] note that the book cards have a different width with the main top bar holding the user photo and main title. please fix that so it follows the zeplin design. 
- [ ] please kindly align the add book button with the book cards and the user photo place holder. nice work
yours
![Screenshot_48](https://github.com/Kwamay/React-Books/assets/65298048/af96f909-229b-489d-8e24-237555d2ae18)
original
![Screenshot_49](https://github.com/Kwamay/React-Books/assets/65298048/c9db9810-928f-42c5-8419-fcdd9273516f)